platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_serial_getty_monitor

```diff
@@ -1,4 +1,5 @@
 #Dell S6100 serial getty monitor
-check process serial-getty matching "ttyS"
+check program serial-getty with path /usr/local/bin/check-getty.sh
```

Comment:
This issue is not specific to S6100, we see in Arista as well

Reply:
Maybe we can fix S6100 script first then porting this to Arista?
I will check arista issue later.

Reply:
Update: I check the Arista issue, it's because some using stuck on console, so others can't login via console. it's not a serial-ketty down issue.

Reply:
I guess there is some typo in "some using stuck". @liuh-80  Could you check?

Reply:
Fixed, 'some user operation stuck on console'